packages/@webex/plugin-meetings/src/locus-info/embeddedAppsUtils.js

```diff
@@ -30,11 +30,18 @@ EmbeddedAppsUtils.parseApp = (embeddedApp) => {
  * @returns {boolean} true if the arrays are different
  */
 EmbeddedAppsUtils.areSimilar = (apps1, apps2) => {
+  if (!apps1 || !apps2) {
+    return apps1 === apps2;
+  }
+
   if (apps1?.length !== apps2?.length) {
     return false;
   }
-  if (apps1?.[0]?.state !== apps2?.[0]?.state) {
-    return false;
+
+  for (let i = 0; i < apps1.length; i += 1) {
+    if (apps1[i].state !== apps2[i].state) {
```

Comment:
I am assuming this will trigger for differnt app but you are checking if its slido or not later

Reply:
Yes, I'm checking in cantina.  I was hoping not to put anything slido-specific in the sdk.